docs/core/install/includes/linux-install-30-dnf.md

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
I'm a little concerned about the number of redundancies with these includes. The only real differences between most of them is the version being installed or the install tool. Would it make more sense to have one top-level doc with the structure and leading paragraphs, with some sort of tabs or zone pivot mechanism?